ONSdigital/response-operations-ui
# Us053 remove loaded sample

Works alongside:

https://github.com/ONSdigital/ras-party/pull/131
https://github.com/ONSdigital/rm-collection-exercise-service/pull/60

This PR enables an internal user to unlink a sample from a collection exercise
https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/248611078/US053+INT-+Remove+a+loaded+sample

**How to test:**

Remove a loaded sample from a collection exercise that is not in `ready for live` or `live` state then check to see if the sample is removed.
